clients/fides-js/src/components/fides.css

```diff
@@ -950,6 +950,7 @@ div#fides-banner-inner .fides-privacy-policy {
 @media screen and (max-width: 768px) {
   div#fides-banner {
     padding: 24px;
+    width: 100%;
```

Comment:
This is totally fine, but I want to take a quick look to understand where the 100% _used_ to come from, because this isn't an issue on the current release